### PR TITLE
DVPN-120 - Fix channel group udf call

### PR DESCRIPTION
**Pull request by relud**

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql [hunk 1]
```diff
@@ -108,7 +108,7 @@ flows AS (
     flow_id
   HAVING
     -- NOTE: flows near date boundaries may not meet this condition for all dates
-    "guardian-vpn" IN UNNEST(ARRAY_AGG(DISTINCT service))
+    LOGICAL_OR("guardian-vpn" = service)
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql:111]: more efficient calculation with the same result

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql [hunk 2]
```diff
@@ -206,7 +206,12 @@ SELECT
   pricing_plan,
   plan_name,
   rp_button_view AS vpn_site_hits,
-  mozfun.vpn.channel_group(utm_medium, utm_source, utm_campaign, utm_content) AS channel_group,
+  mozfun.vpn.channel_group(
+    utm_campaign => utm_campaign,
+    utm_content => utm_content,
+    utm_medium => utm_medium,
+    utm_source => utm_source
+  ) AS channel_group,
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql:214]: arguments were out of order, causing channel group to be incorrectly assigned

**Review by relud — COMMENTED** [review]
backfilled via:
```sql
UPDATE
  `moz-fx-data-shared-prod.mozilla_vpn_derived.funnel_product_page_to_subscribed_v1`
SET
  channel_group = mozfun.vpn.channel_group(
    utm_campaign => utm_campaign,
    utm_content => utm_content,
    utm_medium => utm_medium,
    utm_source => utm_source
  )
WHERE
  TRUE
```

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql (hunk 1 above)
Comment (sean-rose) [reply sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql:111]: Good call.  I think it'd read more naturally with the comparison value on the right though.
```suggestion
    LOGICAL_OR(service = "guardian-vpn")
```

File: sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql (hunk 2 above)
Comment (sean-rose) [reply sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql:214]: I'd suggest changing the `channel_group()` call in `channel_group_proportions_v1` to used named parameters as well for consistency.

**Review by sean-rose — APPROVED** [review]

**Pull request merged**